api/src/main/java/io/strimzi/api/kafka/model/balancing/BrokerCapacity.java

```diff
@@ -30,14 +30,15 @@
         builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({"disk", "cpuUtilization", "inboundNetwork", "outboundNetwork", "overrides"})
+@JsonPropertyOrder({"disk", "cpuUtilization", "cpuCores", "inboundNetwork", "outboundNetwork", "overrides"})
 @EqualsAndHashCode
 public class BrokerCapacity implements UnknownPropertyPreserving, Serializable {
 
     private static final long serialVersionUID = 1L;
 
     private String disk;
     private Integer cpuUtilization;
+    private String cpuCores;
```

Comment:
Kubernetes is using just `cpu`. Not `cpuCores`. What is the reason for using different name?

Reply:
It is still different from `cpuUtilization`. And if I understand it right, you use the same format as the `cpu` field in Kubernetes.

@ppatierno @tombentley WDYT?

Reply:
If we can stick with naming and terminology that users already know from builtin Kubernetes features, it's much better imho. My +1 for using `cpu`.